### PR TITLE
Array validation

### PR DESCRIPTION
**Pull request by mgtitimoli**
I was having a look to your code base, and I detected you were using instanceof Array to check if a given param is an Array, I changed to Array.isArray
reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "env": {
-    "node": true,
-    "browser": true
+    "browser": true,
+    "es6": true,
```
Comment (mgtitimoli) [inline .eslintrc:7]: I also added this env to correctly lint es6

**Comment by netgusto** [comment]
Thank you. I was not aware of this problem.
Is `instanceof` safe at all in multiple frames context, then ? Testing for instance of `Date`, as an example ?

**Comment by mgtitimoli** [comment]
np... It should never happen, but anyway, is good to use this approach instead of using `instanceof` at least for Array that we have this helper method. For the other native objects, you could always do the following:

``` javascript
function isInstanceOfNative(obj, constrName) {
    return Object.prototype.toString.call(obj).endsWith(constrName + "]")
}

if (isInstanceOfNative(new Date(), "Date")) {
   console.log("Yeah, is ugly, but this gets executed!");
}
```

**Comment by netgusto** [comment]
ok, thank you !

**Pull request merged**